src/main/java/org/springframework/data/r2dbc/domain/SettableValue.java

```diff
@@ -45,6 +46,19 @@ public SettableValue(@Nullable Object value, Class<?> type) {
 		this.type = type;
 	}
 
+	/**
+	 * Creates a new {@link SettableValue} from {@code value}.
+	 *
+	 * @param value must not be {@literal null}.
+	 * @return the {@link SettableValue} value for {@code value}.
+	 */
+	public static SettableValue from(Object value) {
```

Comment:
This creates an asymmetry I don't like too much. With one argument you have a factory method, with two args you have to use the constructor.

I'd vote for either only factory methods or constructors but not mixing them in the public API.